packages/core/src/lib/testUtils.ts

```diff
@@ -0,0 +1,234 @@
+import { exec } from 'child_process';
```

Comment:
I think it is better to have comments on functions to explain what they do. Like the `fund` function, when you look at this function, I would not know whether it is BTC fund or EVM fund or both